presidio-analyzer/presidio_analyzer/predefined_recognizers/il_id_recognizer.py

```diff
@@ -0,0 +1,28 @@
+from presidio_analyzer import Pattern
+from presidio_analyzer import PatternRecognizer
+# pylint: disable=line-too-long,abstract-method
+
+# Note: this recognizer is very simple and act mainly as an
```

Comment:
if these are simple recognizers, is it worth committing them as code? can't they be created as "custom recognisers" and then added as documentation?

Reply:
both this and ILDomainRecognizer

Reply:
We mainly give this as an example, not as a feature. Putting them in JSON would likely be harder to figure out IMO.